app/views/content/landing/if-you-have-a-degree.md

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
should we include `train-to-be-teacher` in the URL @EllieNodder ?

I'm conscious that we are building out a lot of landing pages and we may need to make sure it is really clear which is which - let's discuss?